diff -Naur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2015-01-24 15:45:52.000000000 +0300
+++ b/CMakeLists.txt	2015-08-12 15:24:09.495491932 +0300
@@ -23,8 +23,11 @@
 #
 
 include(FindPkgConfig)
+
 pkg_check_modules( OpenGL REQUIRED gl )
 
+find_package(OpenGL REQUIRED)
+
 #
 # Linux needs X11
 #
@@ -38,26 +41,44 @@
 include_directories( ${PROJECT_SOURCE_DIR}/include )
 
 add_library(GLEW_static STATIC src/glew.c )
-add_library(GLEW_shared SHARED src/glew.c )
+# add_library(GLEW_shared SHARED src/glew.c )
 set_target_properties(GLEW_static PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME} PREFIX lib)
-set_target_properties(GLEW_shared PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME} PREFIX "${DLL_PREFIX}")
-target_link_libraries(GLEW_shared ${OpenGL_LDFLAGS})
+# set_target_properties(GLEW_shared PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME} PREFIX "${DLL_PREFIX}")
+# if(MinGW)
+#     target_link_libraries(GLEW_shared ${OPENGL_LIBRARIES})
+# else()
+#     target_link_libraries(GLEW_shared ${OpenGL_LDFLAGS})
+# endif()
 
 add_library(GLEW_MX_static STATIC src/glew.c )
-add_library(GLEW_MX_shared SHARED src/glew.c )
+# add_library(GLEW_MX_shared SHARED src/glew.c )
 set_target_properties(GLEW_MX_static PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME}mx COMPILE_FLAGS "-DGLEW_MX" PREFIX lib)
-set_target_properties(GLEW_MX_shared PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME}mx COMPILE_FLAGS "-DGLEW_MX" PREFIX "${DLL_PREFIX}")
-target_link_libraries(GLEW_MX_shared ${OpenGL_LDFLAGS})
-
-add_executable(glewinfo src/glewinfo.c)
-target_link_libraries(glewinfo GLEW_shared ${OpenGL_LDFLAGS})
+# set_target_properties(GLEW_MX_shared PROPERTIES OUTPUT_NAME ${GLEW_LIB_NAME}mx COMPILE_FLAGS "-DGLEW_MX" PREFIX "${DLL_PREFIX}")
+# if(MinGW)
+#     target_link_libraries(GLEW_shared ${OPENGL_LIBRARIES})
+# else()
+#     target_link_libraries(GLEW_shared ${OpenGL_LDFLAGS})
+# endif()
+
+# add_executable(glewinfo src/glewinfo.c)
+# # target_link_libraries(glewinfo GLEW_shared ${OpenGL_LDFLAGS})
+# if(MinGW)
+#     target_link_libraries(GLEW_shared ${OPENGL_LIBRARIES})
+# else()
+#     target_link_libraries(GLEW_shared ${OpenGL_LDFLAGS})
+# endif()
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     target_link_libraries(glewinfo ${X11_LIBRARIES})
 endif(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
 
-add_executable(visualinfo src/visualinfo.c)
-target_link_libraries(visualinfo GLEW_shared ${OpenGL_LDFLAGS})
+# add_executable(visualinfo src/visualinfo.c)
+# # target_link_libraries(visualinfo GLEW_shared ${OpenGL_LDFLAGS})
+# if(MinGW)
+#     target_link_libraries(GLEW_shared ${OPENGL_LIBRARIES})
+# else()
+#     target_link_libraries(GLEW_shared ${OpenGL_LDFLAGS})
+# endif()
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
     target_link_libraries(visualinfo ${X11_LIBRARIES})
@@ -66,11 +87,11 @@
 install(
 	TARGETS
 		GLEW_static
-		GLEW_shared
+		# GLEW_shared
 		GLEW_MX_static
-		GLEW_MX_shared
-		glewinfo
-		visualinfo
+		# GLEW_MX_shared
+		# glewinfo
+		# visualinfo
 	RUNTIME DESTINATION bin
 	LIBRARY DESTINATION lib
 	ARCHIVE DESTINATION lib
