diff -Naur libpng-1.6.18/CMakeLists.txt ../Jupiter/build_android/3rdparty/libpng/ext_libpng-prefix/src/ext_libpng/CMakeLists.txt
--- libpng-1.6.18/CMakeLists.txt	2015-07-23 03:36:41 +0000
+++ ../Jupiter/build_android/3rdparty/libpng/ext_libpng-prefix/src/ext_libpng/CMakeLists.txt	2015-08-25 20:06:03 +0000
@@ -313,12 +313,12 @@
   endif(PNG_SHARED)
 
   if(PNG_STATIC)
-    if(NOT WIN32 OR CYGWIN OR MINGW)
-      get_target_property(BUILD_TARGET_LOCATION ${PNG_LIB_NAME_STATIC} LOCATION_${CMAKE_BUILD_TYPE})
-      CREATE_SYMLINK(${BUILD_TARGET_LOCATION} libpng${CMAKE_STATIC_LIBRARY_SUFFIX})
-      install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libpng${CMAKE_STATIC_LIBRARY_SUFFIX}
-         DESTINATION ${CMAKE_INSTALL_LIBDIR})
-    endif(NOT WIN32 OR CYGWIN OR MINGW)
+    #if(NOT WIN32 OR CYGWIN OR MINGW)
+    #  get_target_property(BUILD_TARGET_LOCATION ${PNG_LIB_NAME_STATIC} LOCATION_${CMAKE_BUILD_TYPE})
+    #  CREATE_SYMLINK(${BUILD_TARGET_LOCATION} libpng${CMAKE_STATIC_LIBRARY_SUFFIX})
+    #  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libpng${CMAKE_STATIC_LIBRARY_SUFFIX}
+    #     DESTINATION ${CMAKE_INSTALL_LIBDIR})
+    #endif(NOT WIN32 OR CYGWIN OR MINGW)
  endif()
 endif()
 
@@ -328,9 +328,9 @@
 endif()
 if(NOT SKIP_INSTALL_EXECUTABLES AND NOT SKIP_INSTALL_ALL )
   if(NOT WIN32 OR CYGWIN OR MINGW)
-    install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/libpng-config DESTINATION bin)
-    install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/${PNGLIB_NAME}-config
-            DESTINATION bin)
+    #install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/libpng-config DESTINATION bin)
+    #install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/${PNGLIB_NAME}-config
+    #        DESTINATION bin)
   endif(NOT WIN32 OR CYGWIN OR MINGW)
 endif()
 
@@ -343,10 +343,10 @@
   install(FILES png.5                    DESTINATION ${PNG_MAN_DIR}/man5)
   # Install pkg-config files
   if(NOT WIN32 OR CYGWIN OR MINGW)
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libpng.pc
-            DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
-    install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/libpng-config
-            DESTINATION bin)
+    #install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libpng.pc
+    #        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+    #install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/libpng-config
+    #        DESTINATION bin)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PNGLIB_NAME}.pc
             DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
     install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/${PNGLIB_NAME}-config
